test/foundry/EscrowIssuer.t.sol

```diff
@@ -97,4 +97,15 @@ contract EscrowIssuerTest is Test {
         vm.prank(user);
         escrowIssuer.redeemEscrow4YR(10);
     }
+
+    /**
+        test when non-treasury tries to call issue redeemable
+        should revert
+     */
+    function testIssueRedeemableNonTreasury() public {
+        vm.startPrank(user);
+        kwenta.approve(address(escrowIssuer), 10);
```

Comment:
are you sure this works? user doesn't hold kwenta tokens right?

Reply:
Yes it throws the correct error. But I also just now added the specific error to expectRevert() for extra safety